packages/dev-utils/src/web3_factory.ts

```diff
@@ -19,16 +19,22 @@ export interface Web3Config {
     hasAddresses?: boolean; // default: true
     shouldUseInProcessGanache?: boolean; // default: false
     rpcUrl?: string; // default: localhost:8545
+    shouldUseFakeGasEstimate?: boolean; // default: true
 }
 
 export const web3Factory = {
     getRpcProvider(config: Web3Config = {}): ProviderEngine {
         const provider = new ProviderEngine();
         const hasAddresses = _.isUndefined(config.hasAddresses) || config.hasAddresses;
+        config.shouldUseFakeGasEstimate =
+            _.isUndefined(config.shouldUseFakeGasEstimate) || config.shouldUseFakeGasEstimate;
```

Comment:
We should probably have `shouldUseFakeGasEstimate` default to `false`. Currently it defaults to `true`.

Reply:
I think we should actually keep it as defaulting to `true` until we investigate the effects of using the Ganache gas estimator in the near future. It might turn out that we don't need the fake gas estimator anymore, but historically it's always been on. Does that sound okay?

Reply:
Yep, makes sense!